client/src/app/app.module.ts

```diff
@@ -3,14 +3,34 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MyNavComponent } from './my-nav/my-nav.component';
+import { MyDashComponent } from './my-dash/my-dash.component';
+import { LayoutModule } from '@angular/cdk/layout';
+import { MatToolbarModule, MatButtonModule, MatSidenavModule, MatIconModule, MatListModule } from '@angular/material';
+import { MyOwnCustomMaterialModule } from './material';
+import { Routes, RouterModule } from '@angular/router'
+
+const appRoutes: Routes= [
+  {path: '', component: MyDashComponent}
+]
```

Comment:
This should go in the AppRoutingModule, all components should be there in the `app-routing.module.ts` file.